web/src/app/components/PageLayout/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+"use client"
+
+export default function Footer() {
+    return (
+        <div className="flex px-6 py-4 mb-2">
+            <nav className="flex justify-between w-full">
+                <a href="https://github.com/YingLiang2/Battletype" target="_blank" rel="noopener noreferrer">
```

Comment:
can you link official repo instead of your fork

Reply:
Mine is the official repo, because it has the deployment

Reply:
lol what

Reply:
like you can see all the frontend deployments on mine

Reply:
We shouldn't really have our deployments in separate versions of the repo anyway. If anything we should move both deployments to the main repo so that server and client versions don't get out of sync.

Long term, the web app and game server should probably go on the same hosting provider so that 1 deployment = 1 version of the website. 

Either way, it's really weird to have the actual website link to a forked version of the main repo

Reply:
Not really, we'll be putting our githubs on our resume so its not a big chance they'll click it on the main website.

We could do 1 deployment of website, but separatation of concerns is also good because what if vercel or heroku goes down.

Reply:
If vercel or heroku goes down, then the website won't work anyway, so that's not really a benefit. Moving them to the same hosting provider isn't really a big priority to me though.

however, it's definitely better to put the deployment of both vercel and heroku on the main repo. If your fork wasn't up to date, then you'd be deploying an old version of the website. 

why not just have one repo that is the source of truth for our whole project? you can link your own github on your resume if you want

Reply:
My fork won't be out of date anyway, I can always setup a github action to sync it automatically.